server.py

```diff
@@ -0,0 +1,249 @@
+import datetime
+
+from sqlalchemy.sql.expression import null
+import json
+from google.protobuf.json_format import MessageToJson, Parse
+
+#import the generated class
+from proto import user_pb2
+from proto import user_pb2_grpc
+from sqlalchemy import create_engine
+
+# import logger
+from log.logging_func import logger
+
+from model.user_model import User
+from controller.functions import PBtimeToPtime, PtimeToPBtime
+
+from dateutil.relativedelta import relativedelta
+
+from protoc_gen_validate.validator import validate, ValidationFailed
+
+
+class UserServicer(user_pb2_grpc.UserServicer):
+    def __init__(self, userController) -> None:
+        super().__init__()
+        self.UserController = userController
+
+    def CreateUser(self, request, context):
+        logger.info(f'{request.first_name} at {request.email} from state id {request.state_id} tried to register ')
```

Comment:
This sounds better, `Received CreateUser request. email: {request.email}`. If someone new tp the project looks at your log, it's impossible for him to know which method is called